executor/analyze.go

```diff
@@ -526,6 +526,14 @@ func (e *AnalyzeColumnsExec) buildStats(ranges []*ranger.Range) (hists []*statis
 	return hists, cms, nil
 }
 
+var (
+	fastAnalyzeCounterSample        = metrics.FastAnalyzeCounter.WithLabelValues("sample")
+	fastAnalyzeCounterAccessRegions = metrics.FastAnalyzeCounter.WithLabelValues("access_regions")
+	fastAnalyzeCounterRegionError   = metrics.FastAnalyzeCounter.WithLabelValues("region_error")
+	fastAnalyzeCounterSeekKeys      = metrics.FastAnalyzeCounter.WithLabelValues("seek_keys")
+	fastAnalyzeCounterScanKeys      = metrics.FastAnalyzeCounter.WithLabelValues("scan_keys")
```

Comment:
We need to put all the these to the `metrics` directory and register them.

Reply:
The `FastAnalyzeCounter` has been registered?